src/molecules/gv-table.js

```diff
@@ -545,8 +544,9 @@ export class GvTable extends withResizeObserver(LitElement) {
                     (option) => option,
                     (option) => {
                       const style = typeof option.style === 'function' ? option.style(item) : option.style;
-                      return html`<div class="cell" style="${ifDefined(style)}">
-                        ${this._renderCell(option, item, itemIndex)}${this._renderTag(option, item)}${option.icon
+                      const value = option.field ? this._getDataFromField(item, option.field) : '';
```

Comment:
This will not handle all cases, I think for consistency we should extract this line + https://github.com/gravitee-io/gravitee-ui-components/pull/454/files#diff-9451a5b765c3cd497599abddeb499290c1b387c9c9861c50ede312a8c8940393R478-R487 in a private method to be sure everything used to "read/parse" the value is grouped together

Reply:
👍🏻 
Done